### ISSUE TITLE
Working at specific hours

### ISSUE THREAD

**Issue by jacob-jakubiak** [prashantgupta24/automatic-mouse-mover#40]
Could you add some feature that would turn it of at some hour, eg. 6pm?

**Comment by prashantgupta24** [comment]
Yeah that's a good suggestion, although I don't have any dev time available these days. Open to pull requests though!
